<?xml version="1.0" encoding="UTF-8"?><testrun name="JedisTest.testJedis" project="fire-content-service" tests="1" started="1" failures="0" errors="1" ignored="0">
  <testcase name="testJedis" classname="com.fire.jedis.JedisTest" time="1.932">
    <error>redis.clients.jedis.exceptions.JedisConnectionException: java.net.SocketTimeoutException: connect timed out&#13;
	at redis.clients.jedis.Connection.connect(Connection.java:155)&#13;
	at redis.clients.jedis.BinaryClient.connect(BinaryClient.java:83)&#13;
	at redis.clients.jedis.Connection.sendCommand(Connection.java:94)&#13;
	at redis.clients.jedis.BinaryClient.set(BinaryClient.java:100)&#13;
	at redis.clients.jedis.Client.set(Client.java:29)&#13;
	at redis.clients.jedis.Jedis.set(Jedis.java:65)&#13;
	at com.fire.jedis.JedisTest.testJedis(JedisTest.java:23)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)&#13;
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)&#13;
	at java.lang.reflect.Method.invoke(Method.java:606)&#13;
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)&#13;
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)&#13;
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)&#13;
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)&#13;
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)&#13;
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)&#13;
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)&#13;
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)&#13;
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)&#13;
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)&#13;
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)&#13;
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:678)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)&#13;
Caused by: java.net.SocketTimeoutException: connect timed out&#13;
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)&#13;
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:85)&#13;
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:339)&#13;
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:200)&#13;
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:182)&#13;
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)&#13;
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)&#13;
	at java.net.Socket.connect(Socket.java:579)&#13;
	at redis.clients.jedis.Connection.connect(Connection.java:149)&#13;
	... 29 more&#13;
&#13;
</error>
  </testcase>
</testrun>
